packages/server/modules/comments/graph/resolvers/comments.js

```diff
@@ -1,6 +1,8 @@
-const { authorizeResolver, pubsub } = require('@/modules/shared')
+const { pubsub } = require('@/modules/shared')
 const { ForbiddenError, ApolloError, withFilter } = require('apollo-server-express')
+const { Forbidden } = require('@/modules/shared/errors')
```

Comment:
The common convention is to always suffix all error subclasses with `Error`. I realize that there already is a `ForbiddenError` in this scope, but then you should just rely on import aliases (`import {ForbiddenError as OtherForbiddenError} from 'whatever'`)